guides/source/testing.md

```diff
@@ -1028,7 +1028,6 @@ You should test for things such as:
 * was the web request successful?
 * was the user redirected to the right page?
 * was the user successfully authenticated?
-* was the correct object stored in the response template?
```

Comment:
I think removing this is the wrong answer. `assigns` checks the instantied controller ivars, but the wording of this is still a little off. Can we change it to `* was the correct information displayed in the response?`?

Reply:
Hey @gmcgibbon. Perfect. I think it makes total sense!. I updated the commit with your suggestion. But one question:

Should it be:
```
* was the correct information displayed in the response?
```

or maybe

```
* was the correct information displayed in the view?
```
?

Because of the next line uses `in the view` instead of `in the response`
```
* was the appropriate message displayed to the user in the view?
```

What do you think?

thanks 🍻